lsp/nls/src/cache.rs

```diff
@@ -93,7 +94,7 @@ impl CacheExt for Cache {
                 // The unwrap is safe here because (1) we have linearized `file_id` and it must be
                 // in the `lin_registry` and (2) every resolved import has a corresponding position in
                 // the linearization of the file that imports it.
-                let pos = lin_registry.map.get(&file_id).and_then(|lin| lin.import_locations.get(&id)).unwrap();
+                let pos = lin_registry.map.get(&file_id).and_then(|lin| lin.import_locations.get(&id)).unwrap_or(&TermPos::None);
```

Comment:
A random panic was happening here sometimes when imports were invalid, and we're already in the process of creating an import error here.

Reply:
Yeah, we still have a lot of unwraps floating around in `nls` :disappointed: